livechat/agent/web/client.py

```diff
@@ -985,3 +992,28 @@ def add_user_to_chat(self,
             payload = prepare_payload(locals())
         return self.session.post(f'{self.api_url}/add_user_to_chat',
                                  json=payload)
+
+
+# Other
+
+    def send_typing_indicator(self,
+                              chat_id: str = None,
+                              visibility: str = None,
+                              is_typing: bool = None,
+                              payload: dict = None) -> requests.Response:
+        ''' Sends typing indicator.
+
+            Args:
+                chat_id (str): Id of the chat that to send the typing indicator to.
```

Comment:
probably a doc issue?
```suggestion
                chat_id (str): ID of the chat that to send the typing indicator to.
```